src/catalog/tests/snapshots/open__initial_snapshot.snap

```diff
@@ -1238,6 +1238,7 @@ Snapshot {
             ),
         },
     },
+    network_policies: {},
```

Comment:
IIRC we're going to add the initial default Network Policy in a follow up PR?

Reply:
yup